client/components/stats-date-control/stats-date-control-picker.tsx

```diff
@@ -67,9 +71,20 @@ const DateControlPicker = ( {
 		onShortcut( shortcut );
 	};
 
+	const togglePopoverVisibility = () => {
+		const event_from = isOdysseyStats ? 'jetpack_odyssey' : 'calypso';
+
+		if ( popoverOpened ) {
```

Comment:
For now, the event trigger when changing from open->closed. Maybe we want to track the opposite direction?

Reply:
Right, that's what I was aiming for. Thanks for catching it.